core_backend/app/llm_call/check_output.py

```diff
@@ -146,7 +160,7 @@ async def _get_llm_align_score(align_score_data: AlignScoreData) -> AlignmentSco
     try:
         alignment_score = AlignmentScore.model_validate_json(result)
     except ValidationError as e:
-        logger.error(f"LLM alignment score respone is not valid json: {e}")
+        logger.error(f"LLM alignment score response is not valid json: {e}")
 
     logger.info(f"LLM Alignment result: {alignment_score.model_dump_json()}")
```

Comment:
BTW, this line just repeats the llm output log, should we delete it?
<img width="871" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/b546b150-2923-4e3d-a8fe-84d8cc1a7a3b">

Reply:
I think the first one is from lite_llm logging and we may turn that off at some stage. And the second one is from us. 
My vote is to let the both stay.